docs/interact/cli.mdx

```diff
@@ -3,37 +3,74 @@ title: Using the NEAR CLI
 sidebar_position: 2
 ---
 
-Calimero works with all the NEAR tooling including the NEAR CLI.
+Calimero works with all the NEAR tooling including the NEAR CLI. 
 
 You can install the NEAR CLI using: 
+
 ```
 npm install -g near-cli
 ```
 
+You can always check your current version by running:
+
+```
+near --version  # version info appears on the last line of output
+```
+
+:::info
 All methods from NEAR CLI are compatible with Calimero Private Shards and the documentation can be found on the official NEAR docs: https://docs.near.org/tools/near-cli.
+:::
 
 
-To use NEAR CLI with Calimero SDK you will need to:
+## Requirements
+
+To use NEAR CLI with Calimero you will need to:
 - generate an Authentication token on the Security page inside the Calimero Console
 - set the custom RPC endpoint for the CLI tool to use from your Dashboard
 - set the API token for the CLI to use
 
 
-In case you are using a mainnet private shard, set the `NEAR_CLI_MAINNET_RPC_SERVER_URL` environment variable. On the other hand, for testnet use the
-`NEAR_CLI_TESTNET_RPC_SERVER_URL` environment variable instead.
+## Set API key
+
+You can interact with the Calimero shard using `near set-api-key` by running:
+
+```
+ near set-api-key https://rpc.testnet.near.org <AUTH_TOKEN>   
+```
+
+## NEAR state
+
+`near state` shows details of an account's state. To use, run the following command
+
+```
+near state <MAIN_ACCOUNT_ID> --nodeUrl https://api.calimero.network/api/v1/shards/<SHARD_ID>/neard-rpc/ --networkId <SHARD_ID>
+```
+
+:::info
+More detailed information on this can be found in the [setup near cli](/getting_started/generate_token#set-up-the-near-cli-to-access-the-shard-via-cli) docs.
+:::
+
+## RPC Server URL
+
+The RPC server URL allows you to communicate directly with the NEAR network. 
 
+If you are using a mainnet private shard, set the `NEAR_CLI_MAINNET_RPC_SERVER_URL` environment variable.
 
 Mainet Example :
 ```bash
 export NEAR_CLI_MAINNET_RPC_SERVER_URL=<put_your_rpc_server_url_here>
 near set-api-key $NEAR_CLI_MAINNER_RPC_SERVER_URL <api-key>
 ```
 
+On the other hand, for testnet use the `NEAR_CLI_TESTNET_RPC_SERVER_URL` environment variable instead.
```

Comment:
This section contradicts the `Set API key` section.
The variable doesn't need to be called this way, too.

You can make a note of this (that instead of putting `https://rpc.testnet.near.org`, the user needs to put the URL of the shard RPC node.